### ISSUE TITLE
ShyLU: error compiling Albany with Tpetra deprecated code off and FROSch

### ISSUE THREAD

**Issue by jewatkins** [trilinos/Trilinos#10202]
## Bug Report
@trilinos/shylu 

### Description
There's an error building Albany with Tpetra deprecated code off due to a recent change.
```
/home/projects/albany/nightlyCDashTrilinosBlake/build-intel/TrilinosSerialInstall/include/FROSch_Tools_def.hpp(166): error: no instance of overloaded function "Tpetra::Distributor::doPostsAndWaits" matches the argument list
```
https://sems-cdash-son.sandia.gov/cdash/viewBuildError.php?buildid=29552

It's a similar issue to #10171, see https://github.com/trilinos/Trilinos/pull/10164 for more details on the change.

**Comment by cgcgcg** [comment]
@iyamazaki @searhein

**Comment by iyamazaki** [comment]
Thank you for the report @jewatkins, @cgcgcg !! I have been building FROSch without deprecated codes, but not with [#10164](https://github.com/trilinos/Trilinos/pull/10164). I now see the compile errors with the change pulled in.

By the way, I also get compile errors from BDDC without deprecated codes (have you seen these?). I have fixed previous compile errors, but was not sure if I should push since my BDDC tests fail even before my changes.. @srajama1..

**Comment by ikalash** [comment]
Tagging myself so I get updates on this issue: @ikalash.

**Comment by ikalash** [comment]
It looks like I need to implement a similar fix having to do with ``doPostOpsAndWaits" for the DTK library, which is needed for Albany-LCM.  It appears that the first and last arguments to this routine now need to be of type GOView.  How should GOView be defined for the case of a code that is not designed to work with KokkosNodes other than the Serial one?

**Comment by ikalash** [comment]
Also when will the deprecated code in Tpetra be removed?  I know it was originally 2/8 but presumably it has been pushed back?

**Comment by ccober6** [comment]
So the minor release was original slated for 2/8, but it got pushed back 2/15, and then we have had some hardware failures.  We are waiting on three PRs to go through and dev2master pass on them before tagging the minor release (hopefully in the near future).

The minor release date is only the date to get code "marked" for deprecation.  Packages cannot actually remove it until March 22th to give everyone, including applications, a chance to identify dependence on deprecated code and get it fixed.

**Comment by iyamazaki** [comment]
I think you should be able to use the Kokkos HostSpace for these views?

**Comment by iyamazaki** [comment]
[PR#10215](https://github.com/trilinos/Trilinos/pull/10215) has been merged. Please let us know if that solved this issue, @jewatkins!

**Comment by jewatkins** [comment]
@iyamazaki This is fixed, thanks! Will post a new issue for the ROL error we're seeing.
